arch/xtensa/src/esp32s3/Kconfig

```diff
@@ -11,24 +11,24 @@ choice
 	prompt "ESP32-S3 Chip Selection"
 	default ARCH_CHIP_ESP32S3WROOM1
 
-config ARCH_CHIP_ESP32S3WROOM1
-	bool "ESP32-S3-WROOM-1"
-	select ESP32S3_FLASH_4M
-	---help---
-		Generic module with an embedded ESP32-S3.
-
-config ARCH_CHIP_ESP32S3WROOM2
-	bool "ESP32-S3-WROOM-2"
-	select ESP32S3_FLASH_16M
-	select ESP32S3_PSRAM_8M
-	---help---
-		Generic module with an embedded ESP32-S3.
-
-config ARCH_CHIP_ESP32S3MINI1
-	bool "ESP32-S3-MINI-1"
-	select ESP32S3_FLASH_8M
-	---help---
-		Generic module with an embedded ESP32-S3.
+	config ARCH_CHIP_ESP32S3WROOM1
+		bool "ESP32-S3-WROOM-1"
+		select ESP32S3_FLASH_4M
+		---help---
+			Generic module with an embedded ESP32-S3.
+
+	config ARCH_CHIP_ESP32S3WROOM2
+		bool "ESP32-S3-WROOM-2"
+		select ESP32S3_FLASH_16M
+		select ESP32S3_PSRAM_8M
+		---help---
+			Generic module with an embedded ESP32-S3.
+
+	config ARCH_CHIP_ESP32S3MINI1
+		bool "ESP32-S3-MINI-1"
+		select ESP32S3_FLASH_8M
+		---help---
+			Generic module with an embedded ESP32-S3.
```

Comment:
I explored a lot of Kconfig files and see that most of those does not have special alignment inside the `choice` section. I think it is better to remove extra TABs in other places rather than adding here.

Reply:
Ok, I've reverted the changes related to the alignment of `choice` options. I'll leave them out of this PR.